src/openforms/js/components/admin/form_design/utils.js

```diff
@@ -156,6 +158,65 @@ const getPathToComponent = (configuration, key) => {
   return '';
 };
 
+// Determine the translations for a configuration, and transform them into the proper shape
+const extractTranslationsFromConfiguration = configuration => {
+  let componentTranslations = {};
+  if (configuration.components) {
```

Comment:
use the `FormioUtils.eachComponent` - as `components` is not the only key containing nested components, but also `columns`, `rows`...